test/replutil.jl

```diff
@@ -227,8 +227,9 @@ let err_str,
     i = reinterpret(EightBitType, 0x54),
     j = reinterpret(EightBitTypeT{Int32}, 0x54)
 
-    err_str = @except_str Symbol() MethodError
-    @test contains(err_str, "MethodError: no method matching Symbol()")
+    #PR 16130 (Symbol)
+    #err_str = @except_str Symbol() MethodError
+    #@test contains(err_str, "MethodError: no method matching Symbol()")
```

Comment:
how about using some other type that doesn't have a 0-argument constructor? that would capture the point of this test